### ISSUE TITLE
Controller Bugs and Scheme

### ISSUE THREAD

**Issue by ghost** [unc-comp580-comp585/bubble-math#13]
Fix controller bugs and decide on controller scheme

**Comment by bowerscd** [comment]
on a local branch I have.

**Comment by bowerscd** [comment]
fix: 

```javascript

    function process_gamepad_buttons() {
        if (Globals.gamepad.justPressed(Phaser.Gamepad.XBOX360_A) && cursor != -1) {
                lock_in_answer();
        }
    }

```

to 

```javascript

    function process_gamepad_buttons() {
        if (Globals.gamepad.justPressed(Phaser.Gamepad.XBOX360_A, 20) && cursor != -1 && !won) {
                lock_in_answer();
        }
    }

```

**Comment by bowerscd** [comment]
fixed in 4ff9931
